### ISSUE TITLE
Add Useful Articles to Wiki

### ISSUE THREAD

**Issue by EmpireWorld** [UnityCommunity/UnityLibrary#16]
Hi.

We need to add some useful tutorials and articles to Wiki to make it even more useful.

Also we can collect useful posts, articles from the internet and unity community and gather all of them all in one wiki.

A good resource is the [:older_man: Unity Official Wiki](http://wiki.unity3d.com/), there are plenty of old utilities and tutorials that can be updated and added to our wiki.

Also we post tips & tricks to the Wiki, Everyone has it's own tricks, what if everyone have all tricks? it will be nice. 🎉 

Thanks.

**Comment by unitycoder** [comment]
unity wiki has been down few days, better grab full copy of the site if it comes up..
(or download from https://web.archive.org/web/*/http://wiki.unity3d.com/ if can)

**Comment by EmpireWorld** [comment]
Good idea. we can make a copy of unity wiki in our wiki.

**Comment by unitycoder** [comment]
backup of unity wiki, if the site goes down
https://www.dropbox.com/s/p3vrovpuefmtmi7/unity3dwiki.zip?dl=0 *snapshot: 5 aug 2017

**Comment by EmpireWorld** [comment]
Nice job, you are awesome!

We should gather useful content of the unity3d wiki and put them in our wiki.

**Comment by unitycoder** [comment]
all snippets and informative posts from him could be collected (text articles could be placed on wiki, ready shaders to repo), lots of details about shaders/effects/rendering stuff:
https://forum.unity.com/search/member?user_id=163285

**Comment by unitycoder** [comment]
unity official wiki is down again, if anyone has time to copy paste some of those scripts here (in github wiki, or in repo, or separate repo?)

**Comment by hasanbayatme** [comment]
Just wanted to add the archive.org link for reference, this seems to be the latest saved version:

https://web.archive.org/web/20201109001348/http://wiki.unity3d.com/index.php/Main_Page

**Comment by unitycoder** [comment]
could be easier to create new repo, unitywikimirror or something,
then use that same hierarchy there.. or any other ideas, to easily clone and find stuff..? github pages as direct html copy?

    Extensions/ - Further extend the functionality of Unity with .NET plugins (suitable for any Unity licence) and native code plugins (for Pro users only). 
    Particle Library/ - Pre-configured particle systems for use in your own projects. 
    Programming/ - Unity programming resources 
    Scripts/ - Everything from general purpose effects to debugging. 
    Shaders/ - Specialised shaders to make your materials more refined. 
    Wizards/ - Scripts to extend the Unity editor.

**Comment by hasanbayatme** [comment]
Yeah, I agree with copy, we could make a copy using gh-pages and it would be much easier than writing them manually on this wiki.

Can use this gem for downloading a copy from archive.org: https://github.com/hartator/wayback-machine-downloader

**Comment by steinbitglis** [comment]
Make sure that you don't breach the license of the wiki-site by redistributing under a different license.
I'm going to change the license of my own stuff though.

Also, the wayback machine didn't seem to direct me to the most recent copy when I started at the top and clicked my way down, but if I used the original direct link to a script, I got the most recent version (sept. this year for a recently updated script).

https://wiki.unity3d.com/index.php/SerializableDictionary

I can currently find such links through DuckDuckGo searches, but their cache will soon be cleared of the whole wiki, so it will be a lot easier if this is done very very soon.
Maybe there's a better way, I don't know.

**Comment by unitycoder** [comment]
yup, i've send question to unity about it. 
(clearly the old threads indicate that the wiki license choice was a mistake, but nobody had time to change it.. but lets wait for answer)

**Comment by unitycoder** [comment]
So cannot clone those scripts here, lets just collect elsewhere.

big fail from Unity, here's the reply:

> Thank you for contacting the Unity customer service team!
> 
> I apologize for not being able to pick this up sooner! We have been experiencing an influx of tickets and haven't been able to get back to our users as fast as we would like to.
> 
> Thank you so much for your interest in the Unity Wiki content. Considering the importance of your request, I had to escalate it internally with the relevant teams.
> 
> After different discussions, our Legal team let us know that unfortunately, due to the legal reasons we can't make the wiki content public again. 
> 
> What happened with wiki.unity3d.com/ is that it was hosted on an open-source platform with outdated information that was no longer maintained and wasn’t legally compliant. We were unable to identify any business team internally who wished to own the site for the purposes of bringing the site into legal and privacy compliance, and as it was not being heavily used internally or externally, the decision was made to deprecate the site since we are a public company and we have to comply with the rules and regulations of public companies. 
> 
> Based on this, the teams took the decision and won't be recreating or moving information to another site and we cannot allow other users to do so. I'm really sorry about this!
> 
> Today, our users would need to visit https://forum.unity.com/ or https://docs.unity3d.com to explore this comprehensive resource for the Unity Editor.
> 
> Again, I'm really sorry about this situation as I know this may not be the answer you were expecting; but please do let me know in case you have any additional questions and I will try my best to help you
